ios/Runner/Info.plist

```diff
@@ -22,7 +22,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(FLUTTER_BUILD_NUMBER)</string>
+	<string>21</string>
```

Comment:
@nmint8m Any reason why we do hardcoding the build number for iOS?

Reply:
I tried running the fastlane command locally which made this change. I will revert it.